### PR TITLE
GitHub: Update Readme for consistency

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Description

As the title.

Preview: https://github.com/0x5bfa/WinUI-Gallery/blob/5bfa/GitHub-ConsistentReadmeHeader/README.md

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by 0x5bfa** [comment]
@niels9001 Hi! Is there a plan to update Dev Home repo README to have this hero?
It looks great!

File: README.md [hunk 1]
```diff
@@ -1,17 +1,16 @@
-# WinUI 3 Gallery 
+![WinUI Gallery Hero Image](assets/hero.png)
 
-[![Build Status](https://dev.azure.com/ms/microsoft-ui-xaml/_apis/build/status%2Fmicrosoft.WinUI-Gallery?branchName=main)](https://dev.azure.com/ms/microsoft-ui-xaml/_build/latest?definitionId=612&branchName=main) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/your/your-project/blob/master/LICENSE)
```
Comment (karkarl) [inline README.md:3]: These badges are missing.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
+<h1 align="center">
+    WinUI 3 Gallery
+</h1>
+<p align="center">
+  <a href="https://apps.microsoft.com/detail/9P3JFPWWDZRC?launch=true&mode=mini">
+    <img src="assets/storeBadge.png" width="200" /></a>
+</p>
 
-![WinUI Screenshot](README_Images/WinUIGalleryLanding.PNG)
+## ⭐️ Accompanying samples for [WinAppSDK](https://github.com/microsoft/WindowsAppSDK)
```
Comment (karkarl) [inline README.md:11]: I think "Accompanying Sample App for WinUI3" needs to be the second line so that there is no confusion between the different repos.

**Review by karkarl — CHANGES_REQUESTED** [review]
Thanks for submitting! Please see inline comments.

**Pull request closed without merging**